### DISCUSSION TITLE
CusdisComments not rendering

### DISCUSSION THREAD

**Discussion by example-user30** [GetPublii/Publii#1388] [Bug Report]
### Operating system

Windows 11

### Publii version

Version: 0.44.4 (build 16557)

### Issue type

Free plugin

### Bug description

Installed the CusdisComments plugin.  I am using the Mono theme, but have tried this with Simple and QF fashion.  The Comments H2 header is displayed, but all the content that is in the iframe from Cusdis flashes on in the browser for a moment then is rendered 0 pixels high.  I have tried changing the iframe to height 100% in the inspector, but that does not help.  

I have tried this in three browsers - Chome, Firefox and Oprea.  The issue might be Cusdis and not Publii.  Just wondering if anyone else is using it or has seen that?

Screen shots of the relevant code in inspector and what the section looks like on my page.
![shot_240222_172127](https://github.com/GetPublii/Publii/assets/156127535/6191e19b-cb89-463f-9224-b699f6573a5c)
![shot_240222_171957](https://github.com/GetPublii/Publii/assets/156127535/0dac0b93-a02f-4378-9765-2935d2a3541a)

P.S. I searched for CusdisComments in the forum here but got no hits. 

Thanks

P.S.S - this is a part of a project I am working on - there is a good chance I will change the discussion plugin so that I can finish the project.  

### Steps to reproduce

The site it public (although it is not a real place or site, just an experiment) - https://bradshawsbiscuits.com.  Just visit and click an article.  Now scroll to the bottom.

**Comment by example-user30** [comment]
I implemented GraphComments and it does not exhibit this behavior.  I think I will test Cusdis Comments on a simple hand coded  HTML website and see what happens.  It might be an issue with the site resources, or it might be an issue with the plugin.  I will report back.

**Comment by example-user30** [comment]
I am back.  The test worked great.  The add-in code did just what it was supposed to do.  Attached is an image of the form and the code that I used.
![shot_240222_190049](https://github.com/GetPublii/Publii/assets/156127535/06935121-0428-45c9-a588-66ab0878055b)
![shot_240222_190447](https://github.com/GetPublii/Publii/assets/156127535/184a4037-108a-4288-b644-eaf5b297f732)

**Comment by bobmitro** [comment]
The iframe's height is zero. However, the Cusdis script is supposed to calculate it, which it currently does not. Additionally, the console throws an error, which suggests an improper configuration of Cusdis rather than a plugin error. By the way, the Mono theme demo also uses Cusdis and does not have this problem. 
I think you should take another look at the plugin settings. Please also attach a screenshot of the settings so we can see what's going on.

**Reply by example-user30** [reply]
Thanks Bob.  I will do that.  I should have checked the console.  Silly me.

**Comment by asalizzoni** [comment]
I know this post is old, but I wanted to comment the solution I found for future eyes on this post. I also had this problem, and what solved it for me was creating a free account on cusdis.com, then going to "Embed Code" and copying the "data-app-id" I then took this value and pasted it in the Cusdis plugin settings on the Publii app where it asked for the App ID.
